### DISCUSSION TITLE
Error running Golang CI Lint

### DISCUSSION THREAD

**Discussion by ALNBAND** [golangci/golangci-lint#2191] [Q&A]
HI,

I've a tiny go project and when I run the linter i got the following:

WARN [runner] Can't run linter goanalysis_metalinter: buildir: failed to load package internal: could not load export data: no export data for "github.com/tetratelabs/proxy-wasm-go-sdk/proxywasm/internal" 

Not sure how to proceed, which options do I have? 
please advice

**Comment by ALNBAND** [comment]
Any idea how we can overcome it?

**Reply by ldez** [reply]
Hello, have you a go.mod file?
